bigquery/tests/unit/test_job.py

```diff
@@ -3376,3 +3498,29 @@ def test_from_api_repr_normal(self):
         self.assertEqual(entry.records_written, self.RECORDS_WRITTEN)
         self.assertEqual(entry.status, self.STATUS)
         self.assertEqual(entry.steps, steps)
+
+    def test_started(self):
+        klass = self._get_target_class()
+
+        entry = klass.from_api_repr({})
+        self.assertEqual(
+            entry.started,
+            None)
+
+        entry.start_ms = self.START_MS
```

Comment:
I think I'd prefer if you went from a datetime to ms here and then verify. I don't want to have to trust the `_datetime_from_microseconds` function to know the test is correct.